.github/workflows/build-system.yml

```diff
@@ -52,14 +52,6 @@ jobs:
           version: 1.3.211.0
           cache: true
 
-      - name: Install Vulkan SDK (Windows)
-        uses: humbletim/setup-vulkan-sdk@v1.2.0
-        if: "contains(matrix.os, 'windows')"
-        with:
-          vulkan-query-version: 1.3.211.0
-          vulkan-components: Vulkan-Headers, Vulkan-Loader
-          vulkan-use-cache: true
-
```

Comment:
Currently CI has two ways of getting Vulkan SDK prior to CMaking anything - the Linux/macOS method which installs a full copy of the SDK to a portable location on the CI VM (which provides those OSes with an interface for Vulkan as well), or the Windows way which creates a quasi-SDK with only the required elements listed.

Since vending the Vulkan Loader and Headers does the _same_ that the CI action does here, I removed the Windows version to stop the LunarG website 502s from blocking us. Linux and macOS remain unchanged